[DOCS] DOC-337 automate updates to the version information displayed in the getting started tutorial.

Rachel-Reverie opened the pull request:
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
- adds script to automate generating `.mdx` snippet containing the expected output of the `great_expectations --version` CLI command formatted as a markdown codeblock.
- updates the getting started tutorial to use the generated version snippet created by the added script being run.
- adds an instance of the file generated by the script so that docusaurus builds won't fail when running `yarn start` or `yarn build` before running the script is included in the actual release pipeline and the snippet is being generated automatically.


After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-


### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.


Thank you for submitting!

Rachel-Reverie committed:
DOC-337
- adds script to automate generating `.mdx` snippet containing the expected output of the `great_expectations --version` CLI command formatted as a markdown codeblock.

Rachel-Reverie committed:
DOC-337
- updates the getting started tutorial to use the generated version snippet created by the script in the previous commit.
- adds an instance of the file generated by the script in the previous commit so that docusaurus builds won't fail when running `yarn start` or `yarn build` before running the script is included in the actual release pipeline.

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=5348&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=5348&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/5348/83700e18/16ecdceb09644d902e7c0baefd95cf9c79c6c928.svg)](https://app.codesee.io/r/reviews?pr=5348&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

Rachel-Reverie committed:
DOC-337
- adds some clarifications in the notes in the script's docstring and in the `update_version_mdx()` method's docstring.

Rachel-Reverie committed:
DOC-337
- typo correction in script's docstring

Rachel-Reverie committed:
DOC-280
- removes the script (its functionality was added to the releaser tool) but retains the update for the Getting Started Tutorial to use the generated `.mdx` snippet

Rachel-Reverie committed:
Merge branch 'develop' into DOCS/DOC-275/DOC-337/automate_updating_version_info_in_tutorial_docs
